### ISSUE TITLE
undefined type of timeout in crystal master

### ISSUE THREAD

**Issue by kostya** [jessedoyle/duktape.cr#43]
```
Error in lib/duktape/src/duktape/sandbox.cr:15: instance variable @timeout of Duktape::Sandbox was inferred to be Nil, but Nil alone provides no information

      @timeout = nil
      ^
```

**Comment by jessedoyle** [comment]
Hey @kostya!

I'm not currently seeing this behaviour locally.

What's your current version of Crystal?

Here's mine (I believe it's the latest published release):

```
> crystal --version
Crystal 0.24.2 (2018-03-10)

LLVM: 5.0.1
Default target: x86_64-apple-macosx
```

**Comment by kostya** [comment]
crystal master

**Comment by kostya** [comment]
fix is define
```crystal
@timeout : Int32 | Int64 | Nil
```

**Comment by jessedoyle** [comment]
Thanks @kostya for the bug report!

I've fixed this issue in version `0.14.1`. See https://github.com/jessedoyle/duktape.cr/pull/44.

**Comment by kostya** [comment]
thx
